.gitignore

```diff
@@ -7,4 +7,9 @@ okta-oidc-tck
 samples-nodejs-express-4
 .DS_Store
 .vscode
-.samples.config.js
\ No newline at end of file
+.samples.config.js
+custom-login/package-lock.json
```

Comment:
Why are we taking out the package lock files?

Reply:
I thought we didn't need to checkin package-lock files. Should I add them back?

Reply:
Running `npm install` outputs a message noting that the `package-lock.json` files should be committed. Lets make sure these aren't ignored/removed.